osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/services/SecretManagementService.java

```diff
@@ -53,6 +54,7 @@
  * by reencrypting the AES-encrypted secrets to RSA.
  */
 @Service
+@Slf4j
```

Comment:
I think this entire file could be out of this PR. I assume this the result of undoing the added log statements?